### PR TITLE
Helm: contributing, running ct lint with docker

### PR DESCRIPTION
**Pull request by krajorama**
#### What this PR does

Tip about running ct lint with docker. I had trouble installing dependencies.

#### Which issue(s) this PR fixes or relates to

N/A

#### Checklist

- [N/A] Tests updated
- [x] Documentation added
- [N/A] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/internal/contributing/contributing-to-helm-chart.md [hunk 1]
```diff
@@ -36,6 +36,12 @@ Install [ct](https://github.com/helm/chart-testing) and run
 ct lint --config operations/helm/ct.yaml --charts operations/helm/charts/mimir-distributed
 ```
 
+Or use Docker without installing dependencies and run
+
+```bash
+docker run --rm -it --volume $(pwd):/data quay.io/helmpack/chart-testing:latest sh -c "ct lint --all --debug --chart-dirs /data/operations/helm/charts --check-version-increment false --config /data/operations/helm/ct.yaml"
```
Comment (dimitarvdimitrov) [inline docs/internal/contributing/contributing-to-helm-chart.md:42]: I think we should just replace the previous instructions. This requires installing less things.

also you don't need the interactive and tty flags

```suggestion
docker run --rm --volume $(pwd):/data quay.io/helmpack/chart-testing:latest sh -c "ct lint --all --debug --chart-dirs /data/operations/helm/charts --check-version-increment false --config /data/operations/helm/ct.yaml"
```

**Review by dimitarvdimitrov — COMMENTED** [review]
I think we should add ct to the build image and just have a make command like we do for all other checks in the repo.

I understand that increases the scope of the PR and am happy if you choose not to do it.

File: docs/internal/contributing/contributing-to-helm-chart.md (hunk 1 above)
Comment (krajorama) [reply docs/internal/contributing/contributing-to-helm-chart.md]: updated command to not run as root and removed `-it`

**Comment by krajorama** [comment]
> I think we should add ct to the build image and just have a make command like we do for all other checks in the repo.
> 
> I understand that increases the scope of the PR and am happy if you choose not to do it.

This is a bit tricky as linting is actually run in grafana/helm-charts workflow. Let's talk about adding to image another time

**Review by dimitarvdimitrov — APPROVED** [review]
LGTM

**Pull request merged**